### PR TITLE
Provide support for go-style string literals on command line args.

### PR DESCRIPTION
**Pull request by spencerkimball**

### CODE REVIEW

**Comment by bdarnell** [comment]
LGTM

**Comment by tschottdorf** [comment]
LGTM

File: server/cli/kv.go [hunk 1]
```diff
@@ -106,16 +120,10 @@ func runPut(cmd *cobra.Command, args []string) {
...
-	// to unquote them using strconv.Unquote.
-
 	// Do not allow system keys to be put.
-	for i := 0; i < len(args); i += 2 {
-		if strings.HasPrefix(args[i], "\x00") {
-			fmt.Fprintf(osStderr, "unable to put system key: %s\n", proto.Key(args[i]))
-			osExit(1)
-			return
-		}
+	unquoted := []string{}
```
Comment (petermattis) [inline server/cli/kv.go]: `var unquoted []string`

**Comment by petermattis** [comment]
LGTM

**Pull request merged**